test/create_db.sh

```diff
@@ -3,6 +3,37 @@ set -o errexit
 cd $(dirname $0)/..
 source test/db-common.sh
 
+# posix compliant escape sequence
+esc=$'\033'"["
+
+function print_heading() {
+  echo
+  echo -e "${esc}0;34;1m"$1"${esc}0m"
+}
+
+function get_migrations() {
+  migrations=( $(find "$dbpath"/migrations -mindepth 1 -maxdepth 1 -not -path '*/\.*'  2> /dev/null | sort) )
```

Comment:
Why not `ls "${dbpath}/migrations/*.sql`? `ls` sorts by name by default.

Reply:
On parsing ls: http://mywiki.wooledge.org/ParsingLs

But yeah, I also didn't follow the advice around find either. I've since replaced it with a simple for blob followed by a `test || continue`